packages/client/src/utils/sidebarItemList.tsx

```diff
@@ -88,6 +88,8 @@ export const listEE = [
     title: 'Apps',
     subPath: 'apps',
     icon: iconApps,
+    stage: 'alpha',
+    hidden: !(window as any).enableApp,
```

Comment:
define it on `global.d.ts` at `@types` folder then doesn't need `as any`

Reply:
Fix. Thanks!